examples/README.md

```diff
@@ -42,3 +42,6 @@ go get sigs.k8s.io/kustomize
  * [remote target](remoteBuild.md) - Building a kustomization from a github URL
  
  * [json patch](jsonpatch.md) - Apply a json patch in a kustomization
+ 
+ * [transformer configs](transfomerconfigs/README.md) - Customize transformer configurations
```

Comment:
typo on in the name of dir `transfomerconfigs` --> `transformerconfigs` (r missing)

Reply:
Oops, just fixed it.